tests/index.js

```diff
@@ -106,6 +106,8 @@ describe('Thorn', () => {
                         name: 'TestRecord1',
                         testField1: 'TestField1data',
                         testField2: 'TestField2data',
+                        testField3: false,
```

Comment:
Since the test file is already pretty big (#142), I'm okay with reusing the existing test.

As a compromise, IMO we should put a comment explaining that we're testing this bug here.